yarn.lock

```diff
@@ -10649,6 +10682,16 @@ react-transition-group@^2.2.1:
     prop-types "^15.6.2"
     react-lifecycles-compat "^3.0.4"
 
+react@^16.5.2:
```

Comment:
ping @inyono is this a problem? Like: Two React versions? Is required by `react-mathquill`